### PR TITLE
chore: prepare the repo for a stable release

### PR DESCRIPTION
**Pull request by KnorpelSenf**
- Delete .tool-versions
- Adds year and plugin author to license
- Adds various missing fields to package.json
- No longer depends on @grammyjs/types directly
- Fixes the Deno-specific README
- Make code blocks in the main repo consistent
- Runs `deno task fix` with latest Deno
- Update gh action release task to latest version
- Updates deps in CI
- Drops useless package-lock.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,8 +15,15 @@
   ],
   "author": "Roz <roz@rjmunhoz.me>",
   "license": "MIT",
+  "homepage": "https://grammy.dev/plugins/commands.html",
```
Comment (KnorpelSenf) [inline package.json:18]: Ah it should be
```suggestion
  "homepage": "https://grammy.dev/plugins/commands",
```

File: src/README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
-## grammY Commands Plugin
+# grammY commands
 
-This plugin provides a convenient way to define and manage commands for your grammY bot. It simplifies the process of
-setting up commands with scopes and localization.
+This commands plugin for [grammY](https://grammy.dev) provides a convenient way to define and manage commands for your grammY bot.
+It simplifies the process of setting up commands with scopes and localization.
+Please confer the [official documentation](https://grammy.dev/plugins/commands) for this plugin to learn more about this plugin.
```
Comment (roziscoding) [inline src/README.md:5]:
```suggestion
Please confer the [official documentation](https://grammy.dev/plugins/commands) for this plugin to learn more about it.
```

**Review by roziscoding — DISMISSED** [review]

**Review by roziscoding — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
Thanks, feel free to merge

**Comment by roziscoding** [comment]
We actually need to sync the READMEs

**Comment by KnorpelSenf** [comment]
Ready from my side.

**Comment by carafelix** [comment]
Thanks for removing the @grammyjs/types dependency, I totally forgot to update that afterwards

**Review by roziscoding — APPROVED** [review]

**Pull request merged**